git_machete/client.py

```diff
@@ -1682,18 +1682,21 @@ def checkout_github_prs(self, pr_no: int) -> None:
         remote: str
         remote, (org, repo) = self.__derive_remote_and_github_org_and_repo()
         debug('checkout_github_pr()', f'organization is {org}, repository is {repo}')
+        print(f"Fetching {remote}...")
+        self.__git.fetch_remote(remote)
+        if self.__git.remotes():
```

Comment:
Hmm actually I've just noticed now... since we've just fetched `remote` (which corresponds to a Github-hosted remote), there is at least one remote... so this check is redundant